xblock/core.py

```diff
@@ -90,6 +99,9 @@ def open_local_resource(cls, uri):
         unauthorized resource.
         """
 
+        if isinstance(uri, six.binary_type):
+            uri = uri.decode('utf-8')
```

Comment:
I think the catch here is that the variable doesn't describe a URI but a *local* resource.